client/blocks/reader-post-card/style.scss

```diff
@@ -28,7 +28,6 @@
 .reader-post-card.card {
 	border-bottom: 5px solid var(--color-neutral-5);
```

Comment:
Just curious, what does this do?

Reply:
On screens < 660px it forms a solid 5px border between the cards, rather than the inset outlined card on desktop.

<img src="https://github.com/user-attachments/assets/fb334b22-f054-4114-a9db-0fb81a58fed6" width="300">